README.md

```diff
@@ -121,9 +124,35 @@ as an array, you can call typical array methods to choose a particular company t
 
 You can find the API information for this endpoint [here](https://github.com/FareHarbor/fareharbor-docs/blob/master/external-api/endpoints.md#companies).
 
-    GET /companies/<shortname>/lodgings/
+    `GET /companies/<shortname>/lodgings/`
+
+Returns a list of all lodgings for a specific company, provided you have permission to access that company
```

Comment:
I don't think we need to specify that you need permission to access the company since we will be stating that earlier in the documentation